examples/kubernetes-releases/kubernetes1.6.json

```diff
@@ -6,7 +6,7 @@
       "orchestratorRelease": "1.6"
     },
     "masterProfile": {
-      "count": 3,
+      "count": 1,
       "dnsPrefix": "",

```

Comment:
OK; I was under the impression these files were examples for end-users to consume rather than test scenarios based on the fact that they were in the examples/ dir.  Might be worth moving them into an appropriately-named directory if they're test scenarios.